numpy/polynomial/_polybase.py

```diff
@@ -284,9 +310,37 @@ def __repr__(self):
         return f"{name}({coef}, domain={domain}, window={window})"
 
     def __str__(self):
-        coef = str(self.coef)
-        name = self.nickname
-        return f"{name}({coef})"
+        # Get configuration for line breaks
+        linewidth = np.get_printoptions().get('linewidth', 75)
+        if linewidth < 1:
+            linewidth = 1
+        out = ""
+        if self.coef[0] < 0:
+            out += "-"
+        out = f"{self.coef[0]} "
+        cur_line = len(out) // linewidth    # For handling linebreaks
+        for i, coef in enumerate(self.coef[1:]):
+            next_term = ""
+            power = str(i + 1)
+            # Determine +/- symbol
+            if coef >= 0:
+                next_term += f"+ {coef}"
+            else:
+                next_term += f"- {np.abs(coef)}"
+            # Handle different bases
+            if self.basis_name is None:
+                next_term += f"x{power.translate(self._superscript_mapping)} "
```

Comment:
It's a bit weird to assume that a lack of basis_name implies a specific subclass.

It might be more consistent to define a `_str_term` to match the `_repr_latex_term`, and put the power series special case in the subclass override of that method

Reply:
Good idea, I'll take a look.

Reply:
I followed your suggestion in 795ceb6. I defined `_str_term` slightly differently than `_repr_latex_term`. since the `__str__` doesn't (yet) support the variable mapping between domain <-> window, I left off the `need_parens` argument.